Sources/xcproj/AEXML+XcodeFormat.swift

```diff
@@ -0,0 +1,137 @@
+import Foundation
+import AEXML
+
+extension AEXMLDocument {
```

Comment:
Excellent approach @ilyapuchka. We resorted to a custom writer for the `.pbxproj` for similar reasons. In that case, it even added some extra comments on the elements.